rotkehlchen/globaldb/handler.py

```diff
@@ -305,13 +305,16 @@ def get_assets_mappings(identifiers: List[str]) -> Dict[str, dict]:
         identifiers_query = f'assets.identifier IN ({",".join("?" * len(identifiers))})'
         with GlobalDBHandler().conn.read_ctx() as cursor:
             cursor.execute(
-                'SELECT assets.identifier, assets.name, common_asset_details.symbol FROM assets '
+                'SELECT assets.identifier, name, symbol, chain FROM assets '
                 'JOIN common_asset_details on assets.identifier = common_asset_details.identifier '
```

Comment:
Btw shouldn't custom assets be taken into account here as well?

I mean as far as I know, in sqlite `JOIN` works like `INNER JOIN`, right? So doing `JOIN` with `common_asset_details` won't return any custom assets, right?

Or are custom assets not needed here? If they are not, I would then adjust functions names / docstings / api docs, to mention it.

Reply:
thanks for pointing it out.

Reply:
You are always welcome :sunglasses: :sunglasses: :sunglasses: